tests/Medidata.ZipkinTracer.Core.Test/ZipkinClientTests.cs

```diff
@@ -569,6 +569,48 @@ public void RecordBinary_IsTraceOnIsFalse_DoesNotAddBinaryAnnotation()
             Assert.AreEqual(0, testSpan.Binary_annotations.Count, "There are annotations but the trace is off.");
         }
 
+        [TestMethod]
+        [TestCategory("TraceRecordTests")]
+        public void RecordLocalComponent_WithNotNullValue_AddsLocalComponentAnnotation()
+        {
+            // Arrange
+            var testValue = "Some Value";
+            var tracerClient = SetupZipkinClient();
+            spanTracerStub = MockRepository.GenerateStub<SpanTracer>(spanCollectorStub, MockRepository.GenerateStub<ServiceEndpoint>(), new List<string>(), fixture.Create<string>(), fixture.Create<string>());
```

Comment:
idk what are the style guidelines in the .Net world but in ruby that line would be considered too long (must at most 120 chars)